src/features/movieDetails/components/TopTileContainer/index.js

```diff
@@ -15,14 +15,15 @@ import {
 
 export const TopTileContainer = ({
 
-    title = "Mulan long title",
-    rate = 7.8,
-    vote = 3876,
+    imageSrc,
+    title,
+    rate,
+    vote,
 }) => {
     return (
         <Wrapper>
             {/* <ShadowContainer> */}
```

Comment:
Shadow container should not be rendered if there is no backdrop picture available in API (Chris talks about it in the video, go back to it)